### PR TITLE
Update the CI pipeline to enforce comment code style

### PR DESCRIPTION
**Pull request by wiwei**
Some of our comments don't have a space between // and the comment (i.e. //thisisthecomment instead of // this is the comment).

It's somewhat tedious to call this out on each review manually, but the inconsistency was triggering some OCD (i.e. seeing some places in the same file that were good about this, and some that weren't). This should add an automatic way of catching it so that folks can just fix their nit level changes by seeing them explicitly in the CI logs.

See a sane coding style for why there should be some consistency and professionalism in how we do this:

https://google.github.io/styleguide/cppguide.html#Comments

### CODE REVIEW

**Comment by wiwei** [comment]
I noticed this showing up in some recent PRs and wanted to make sure that we don't keep adding more, so hopefully this will make it so that we don't do this anymore.

**Comment by wiwei** [comment]
Also, super cognizant that this doesn't take this far enough (i.e. ideally should autofix as well), and if it's not far enough it could be a pain in the butt - we can see if this ends up being the case, so let's keep an open mind.

File: Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs [hunk 1]
```diff
@@ -1765,12 +1762,12 @@ public bool IsItemVisible(int indexOfItem)
 
             if (indexOfItem < numItemsPrevView)
             {
-                //its above the visible area
+                // Its above the visible area
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs:1765]:
```suggestion
                // It's above the visible area
```

File: Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs [hunk 2]
```diff
@@ -1765,12 +1762,12 @@ public bool IsItemVisible(int indexOfItem)
...
                 itemLoc = false;
             }
             else if (indexOfItem > numItemsPostView)
             {
-                //its below the visible area
+                // Its below the visible area
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs:1770]:
```suggestion
                // It's below the visible area
```

**Pull request merged**